content/docs/api-reference/get_url.md

```diff
@@ -63,8 +63,9 @@ an appropriate library, such as [`boto3`] or [`paramiko`].
   walking up from the current working directory tree).
 
 - `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
-  a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
-  option is ignored. _Default_: `HEAD`.
+  a branch or tag name, a commit hash or an
+  [experiment](/doc/command-reference/exp) name). If `repo` is not a Git repo,
```

Comment:
```suggestion
  a branch or tag name, commit hash, or
  [experiment name](https://dvc.org/doc/user-guide/experiment-management/experiments-overview#how-does-dvc-track-experiments)).
  If `repo` is not a Git repo,
```

Alternatively, link to https://dvc.org/doc/command-reference/exp/run#-n instead.

Reply:
Updated in https://github.com/iterative/dvc.org/pull/3613. PTAL when you can guys. Thanks